python/lsst/fgcmcal/fgcmBuildStarsTable.py

```diff
@@ -95,7 +95,9 @@ class FgcmBuildStarsTableConnections(pipeBase.PipelineTaskConnections,
     )
 
     visitSummary = connectionTypes.Input(
-        doc="Per-visit summary statistics table",
+        doc=("Per-visit consolidated exposure metadata.  These catalogs use "
+             "detector id for the id and are sorted for fast lookups of a "
```

Comment:
Because this is Input, I would phrase it as "...and must be sorted for fast ...", since that's a requirement on the files, not something this package guarantees.